network/transparent_vlan_endpointclient_linux.go

```diff
@@ -513,6 +513,13 @@ func (client *TransparentVlanEndpointClient) DeleteEndpointsImpl(ep *endpoint, g
 
 	log.Printf("[transparent vlan] There are %d routes remaining after deletion", routesLeft)
 
+	// Delete Host Veth
+	if deleteHostVeth {
```

Comment:
Is there a condition where this is `false` ?
I am trying to understand that do we need to delete the host veth pair in case of transparent vlan mode. Can we do that as a separate method ?

Because right now we are changing the method attribute to have `deleteHosVeth` and in few cases the hostveth will be deleted irrespective of this bool.(example: [bridge_endpointclient_linux.go](https://github.com/Azure/azure-container-networking/pull/1892/files#diff-95498b34f8c2709d974c09eefa4a170ba97917bdffc7094901ff1492a9766a95)

Reply:
thats fine bridge mode is deprecated. there are other ways but it will add complexity to code. i would prefer this way